packages/core/src/shared/resourcefetcher/httpResourceFetcher.ts

```diff
@@ -133,50 +133,6 @@ export class HttpResourceFetcher implements ResourceFetcher<Response> {
     }
 }
 
-export class RetryableResourceFetcher extends HttpResourceFetcher {
```

Comment:
Nice that we can eliminate this! Generally better to have composable concepts,  or flexible classes, instead of specific subclasses that implement particular compositions.

Reply:
Yeah that was one of the motivations for doing all this httpResourceFetcher stuff